src/Domain/Account/AccountChartId.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpFinance\DoubleEntry\Domain\Account;
+
+final readonly class AccountChartId
```

Comment:
What's the benefit of having a class for it?

Reply:
Users of `php-finance/double-entry` will use this object as ID of account chart.